src/events/preventEmojiSpam.ts

```diff
@@ -0,0 +1,23 @@
+import { Events, Message } from 'discord.js'
```

Comment:
Fix ESLint warning and should be good to go

Reply:
we should make all warnings error and fail the build. otherwise, reviewers will have to keep ask people to remove warnings.

but that's out of scope for this PR.

therefore, gonna merge it for now